package.json

```diff
@@ -59,4 +60,4 @@
   "peerDependencies": {
     "@uniswap/sdk-core": ">=3"
```

Comment:
remove here?

Reply:
sorry ignore

Reply:
this is intentional: it is installed at a fixed version as a devDependency, but the actual versioning is more open for the peerDependency.